nixpkgs.nix

```
@@ -0,0 +1,5 @@
+# nixpkgs-unstable from 2020-04-05
+import (fetchTarball {
```

```suggestion
import (fetchTarball {
  name = "nixpkgs-unstable-2020-04-05";
```

So the file name in `/nix/store/...` describes the contents of the file (i.e. `/nix/store/rrjij4v3r93n77qnraq2n81d1lapixpv-nixpkgs-unstable-2020-04-05`)